src/Azure.AppConfiguration.Emulator.Service/Formatters/Json/KeyValueJsonInputFormatter.cs

```diff
@@ -33,6 +33,7 @@ protected override bool CanReadType(Type type)
         {
             return base.CanReadType(type) &&
                    (typeof(KeyValueFilterModel).IsAssignableFrom(type) ||
```

Comment:
Is KeyValueFilterModel really supposed to be there?

Reply:
I don't think KeyValueFilterModel belongs here. It should be only KeyValueModel.

Also remove this
          SupportedMediaTypes.Insert(0, MediaTypeHeaderValues.KvsApplication);